tests/DoctrineAuditBundle/Manager/AuditManagerTest.php

```diff
@@ -564,6 +564,36 @@ public function testGetHelper(): void
         static::assertInstanceOf(AuditHelper::class, $manager->getHelper(), 'helper instanceof AuditHelper::class');
     }
 
+    public function testRevert(): void
+    {
+        $em = $this->getEntityManager();
+        $configuration = $this->getAuditConfiguration();
+        $helper = new AuditHelper($configuration);
+        $manager = new AuditManager($configuration, $helper);
+        $reader = new AuditReader($configuration, $em);
+
+        $author = new Author();
+        $author
+            ->setId(1)
+            ->setFullname('John Doe')
+            ->setEmail('john.doe@gmail.com');
+
+        $manager->update($em, $author, [
+            'fullname' => ['John Doe', 'Dark Vador'],
+            'email' => ['john.doe@gmail.com', 'dark.vador@gmail.com'],
+        ], 'what-a-nice-transaction-hash');
+
+        $manager->update($em, $author, [
+            'fullname' => ['John Doe', 'Darth Vador'],
+            'email' => ['john.doe@gmail.com', 'darth.vador@gmail.com'],
+        ], 'what-a-nice-transaction-hash');
```

Comment:
1. You should persist and flush `$author` first (this will log 1 "insert" entry in audit table).
2. Then you should change properties (name, email...) using relevant entity setters and persist and flush again (this will log 1 "update" entry in audit table). Repeat this second step as many time as you want.
3. At this point, the reader should be able to retrieve entries